ouroboros-consensus/test-consensus/Test/Dynamic/General.hs

```diff
@@ -220,3 +262,53 @@ prop_general k TestConfig{numSlots, nodeJoinPlan, nodeTopology} schedule
       where
         NumSlots t = numSlots
         NodeJoinPlan m = nodeJoinPlan
+
+    -- check for Chain Growth violations if there are no Common Prefix
+    -- violations
+    --
+    -- We consider all possible non-empty intervals, so the interval span
+    -- @s@ varies but is always at least 1. We compute a different /speed
+    -- coefficient/ @τ@ for each interval under the assumption that there are
+    -- no message delays (ie @Δ = 0@). This is essentially a count of the
+    -- active slots for that interval in the refined @schedule'@.
+    prop_all_growth =
+        (.||.) (not isConsensusExcepected) $
```

Comment:
Using `(.||.)` in prefix notation is a bit cryptic. IMO, infix (albeit with some extra parens in the second operand) would be more readable.

s/..Excepected/..Expected/

Reply:
Done